Mod Menu Helper/assets/modmenu/lang/tr_tr.json

```diff
@@ -3,42 +3,49 @@
+  "modmenu.summaryTranslation.enhancedblockentities": "Daha hızlı sandık, tabela, yataklar vb.\n§9✎§8 Geliştirilecek bloklar, yöntem",
   "modmenu.summaryTranslation.entity_texture_features": "OptiFine'daki rastgele ve ışık emen moblar\n§9✎§8 Rastgele moblar, ışık emiciliği, göz kırpabilmeleri, ...",
-  "modmenu.summaryTranslation.entityculling": "Engellerin arkasındaki mobları görüntüleme\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.entityculling": "Duvarların arkasındaki mobları önizlemeyi durdur\n§8✎ (sadece gelişmiş seçenekler)",
   "modmenu.summaryTranslation.fabrishot": "F9'la yüksek çözünürlüklü görüntü al\n§9✎§8 F2, çözünürlük, tuşlar vb. üzerine yazma",
   "modmenu.summaryTranslation.farsight": "Sunuculardaki yüklü yığınları yüklü tutar\n§8✎ (sadece gelişmiş seçenekler)",
   "modmenu.summaryTranslation.fastchest": "§c*§r Daha hızlı sandıklar\n§9✎§8 Modu aç/kapa",
   "modmenu.summaryTranslation.ferritecore": "RAM kullanımı azaltır\n§8✎ (sadece gelişmiş seçenekler)",
   "modmenu.summaryTranslation.iris": "OptiFine'ın shaderları\n§9✎§8 Modu aç/kapa, shader, tuş atamaları vb. seç",
-  "modmenu.summaryTranslation.itemmodelfix": "Elde tutulan eşyadaki görünmez çizgileri düzeltir\n§9✎§8 Metod",
+  "modmenu.summaryTranslation.itemmodelfix": "Elde tutulan eşyadaki görünmez çizgileri düzeltir\n§9✎§8 Yöntem",
   "modmenu.summaryTranslation.lambdabettergrass": "OptiFine'daki \"daha iyi çimenler\"\n§9✎§8 Kalite, kar, kaynak paketi",
   "modmenu.summaryTranslation.lambdynlights": "OptiFine'ın \"dinamik ışıklandırmaları\"\n§9✎§8 Kalite, varlık seçenekleri, ...",
   "modmenu.summaryTranslation.languagereload": "Hızlı dil değiştirme, arama çubuğu\n§9✎§8 Dili yeniden yüklemek için F3+J",
   "modmenu.summaryTranslation.lazydfu": "Daha hızlı oyun açılışı\n§8✎ (seçenekleri yok)",
   "modmenu.summaryTranslation.lithium": "Fizikleri ve yığınları optimize eder\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.isxander-main-menu-credits": "Mod paketi sürümü başlık ekranında\n§8✎ (sadece gelişmiş seçenekler)",
+  "modmenu.summaryTranslation.memoryleakfix": "Bellek sızıntılarını düzeltir\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.midnightcontrols": "Oyun kumandası ve dokunmatik ekran desteği\n§9✎§8 Giriş yöntemi, tuş atamaları, oyun kumaandası seçenekleri, ...",
+  "modmenu.summaryTranslation.mixin-conflict-helper": "İki mod çakıştığında uyarır\n§8✎ (seçenekleri yok)",
   "modmenu.summaryTranslation.modmenu": "Bu mod menüsü\n§9✎§8 Tuş yerleşimi, filtreleme, sıralama, ...",
+  "modmenu.summaryTranslation.moreculling": "Blokların gizli taraflarını önizlemeyi durdur.\n§9✎§8 Blokları aç/kapa, eşya çerçeveleri, ...",
   "modmenu.summaryTranslation.morechathistory": "Sohbet geçmiş limitini arttırır\n§8✎ (seçenekleri yok)",
+  "modmenu.summaryTranslation.nochatreports": "Seni ve diğerlerini bazı hesap yasaklamalarından korur\n§8✎ (sadece gelişmiş seçenekler)",
```

Comment:
"korur" does not fit to the first row

Reply:
How can I solve this problem? There is no alternative

Reply:
Try "Sizi ve diğerlerini bazı hesap banlamalarından korur\n§8✎ (sadece gelişmiş seçenekler)"
"banlama" should be shorter and OK in a casual context.

Reply:
yasaklama is turkish version of banlama

Reply:
I know that might kinda defeat the purpose of the translation of the original string, but should we make an exception and suggest making whole different sentence that describes the mod without compromising the original context? Since you've translated the No Chat Reports before, maybe we can find a way to construct a shorter and explanatory sentence?

Reply:
Ok I will make this but "çözmek" and "düzeltmek" are not interchangeable

Reply:
I think I found a solution. "yasaklamalarından" has been superseded by "yasaklarından"